repo-scripts/size-analysis/analysis-helper.ts

```diff
@@ -162,19 +147,40 @@ export async function extractDependenciesAndSize(
   fs.unlinkSync(input);
   fs.unlinkSync(externalDepsNotResolvedOutput);
   fs.unlinkSync(externalDepsResolvedOutput);
-  return {
-    dependencies,
-    sizeInBytes: Buffer.byteLength(
-      externalDepsNotResolvedOutputContentMinimized.code!,
-      'utf-8'
-    ),
-    sizeInBytesWithExternalDeps: Buffer.byteLength(
-      externalDepsResolvedOutputContentMinimized.code!,
-      'utf-8'
-    )
+  const exportData: ExportData = {
+    name: '',
+    classes: null,
+    functions: null,
+    variables: null,
+    enums: null,
+    externals: null,
+    size: 0,
+    sizeWithExtDeps: 0
   };
+  exportData.name = exportName;
+  for (const key of Object.keys(dependencies)) {
+    exportData[key] = dependencies[key];
+  }
+  exportData.externals = serializeExternalField(dependencies.externals);
```

Comment:
Can we change `extractExternalDependencies()` to return a map, so we can remove this transformation?